<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1242</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit_rMv">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>30</y>
      <width>161</width>
      <height>330</height>
     </rect>
    </property>
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_bMv">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>30</y>
      <width>151</width>
      <height>330</height>
     </rect>
    </property>
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_red">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>10</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>红方</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_black">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>10</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>黑方</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_undo">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>930</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>悔棋</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_gamePattern">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>370</y>
      <width>310</width>
      <height>40</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>自我博弈</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>人人对战</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>人机对战</string>
     </property>
    </item>
   </widget>
   <widget class="QTextEdit" name="textEdit_msg">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>460</y>
      <width>310</width>
      <height>240</height>
     </rect>
    </property>
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_send">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>880</y>
      <width>310</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>SendMessage</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_send">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>770</y>
      <width>310</width>
      <height>87</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_CR">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>440</y>
      <width>160</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Chatting Records:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_SM">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>740</y>
      <width>150</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Sending Message:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1242</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
